gateway-js/package.json

```diff
@@ -35,6 +35,7 @@
     "@apollo/utils.logger": "^1.0.0",
     "@josephg/resolvable": "^1.0.1",
     "@opentelemetry/api": "^1.0.1",
+    "@types/node-fetch": "2.6.2",
```

Comment:
Why not `^2.6.2`?

Reply:
Because I copied it from the top level package.json and didn't notice. Thanks for the catch.